app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError
```

Comment:
Consider providing a message when raising `ValueError` to give more context about the error. For example, `raise ValueError('Invalid command format')`.